content/blog/2021/05/2021-05-15-configure-plugins-with-jcasc.adoc

```diff
@@ -0,0 +1,114 @@
+---
+:layout: post
+:title: "Configure Plugins with JCasC"
+:tags:
+- jenkins
+- jcasc
+- tutorial
+- plugins
+- yaml
+:author: dheerajodha
+opengraph:
+  image: image:/images/post-images/2021-05-15-configure-plugins-with-jcasc/opengraph-image.jpg
+---
+
+This blog post is for anyone interested to know how to configure a plugin using the Jenkins Configuration as a Code (JCasC) plugin, more specifically, this blog will guide you to get the YAML equivalent of a plugin’s configuration and use it to do some changes to the plugin without using the Jenkins UI.
+
+---
+If you’re a beginner at JCasC and want to learn more about it, you can head over to the following resources to understand JCasC better:
+
+* link:https://plugins.jenkins.io/configuration-as-code/[JCasC Documentation]
+* link:https://www.youtube.com/watch?v=PAKWqRE0aTk[Overview of JCasC (Online Meetup)]
```

Comment:
There should be newer recordings by @ewelinawilkosz , I believe

Reply:
Yes, I found a couple of them:
 [6 months old](https://www.youtube.com/watch?v=wTzljM-EDjI)
 [1+ year old](https://www.youtube.com/watch?v=XMpdhvs-ceU)
 
Which one should I add?

Reply:
I prefer the video that is 6 months old.  https://www.youtube.com/watch?v=wTzljM-EDjI

Reply:
Thank you :+1: